extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -100,6 +103,19 @@ protected function renderItem( $type, array $data ) {
 		return $result;
 	}
 
+	protected function renderData( $nodeData ) {
+		if (
+			!empty( $nodeData['collapse'] ) &&
+			$nodeData['collapse'] === \Wikia\PortableInfobox\Parser\Nodes\NodeData::COLLAPSE_CLOSED_OPTION &&
+			// You can't predict image size before it's loaded, so don't collapse fields with images
+			!$this->getImageHelper()->doesHtmlContainImage( $nodeData['value'] )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "value" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)